humans.txt

```diff
@@ -0,0 +1,43 @@
+/* TEAM */
+
+Co-founder: Alex Jover
+Site: https://alexjoverm.github.io/
+Twitter: @alexjoverm
+Location: Alicante, Spain
+
+Co-founder: Álvaro Yuste
+Site: http://yuste.info/
+Twitter: @AlvYuste
+Location: Alicante, Spain
+
+Co-founder: Aarón García
+Site: https://aarongarciah.com/
+Twitter: @aarongarciah
+Location: Alicante, Spain
+
+Contributor: Yannick Miñano
+Twitter: @leoynick
+Location: Alicante, Spain      
+
+Contributor: Laura Bonmati
+Site: http://www.laurabonmati.com/
+Twitter: @LauraBonmati
+Location: Alicante, Spain                				
+
+							
+/* THANKS */
+
+Speakers: Flavio Corpa, José Manuel Pérez, Joaquín Oltra, Kikobeats, Ciro Ivan.
+
+Sponsors: ULab, Coosto, VueJobs, Fun Academmy. 
```

Comment:
I wouldn't add this info about sponsors because they change over the time. What do you think @AlvYuste @alexjoverm ?

Reply:
As I understand this file will suffer constant modifications as for the contributors. So **current** sponsors could be here also.